### ISSUE TITLE
[BUG] model API not working for Robustness test

### ISSUE THREAD

**Issue by simin-ong** [IMDA-BTG/aiverify#241]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I tried to use the sample datasets provided (tabular binary classification) and followed the 'preparation of input files' to train a simple LogisticRegression model. Thereafter, I deployed the model using FastAPI and tried to use the new AI Model API feature on the user end. However, none of the tests could run.

Do you have a guide on 1) How to setup a Model API to be used in conjunction with AI Verify and 2) How to build a plug-in suitable for use with Model API? 

### Current Behavior

When I run the tests, there are no logs on the API side, suggesting that AI Verify has not reached the stage whereby an API to the model has been called. There might be a bug before this.

I have checked that the URL works.
![image](https://github.com/IMDA-BTG/aiverify/assets/60313531/f5f7d4e1-c335-4b55-954f-1f0cc1b36fc1)

### Steps To Reproduce

This is the API model configuration.
![image](https://github.com/IMDA-BTG/aiverify/assets/60313531/1955b523-57b6-4896-ae5b-4517764d273a)
![image](https://github.com/IMDA-BTG/aiverify/assets/60313531/1d81492f-8e1d-428a-813d-6b66e82a9e6d)

### Environment

```markdown
- Operating System and Version:
- AI Verify Version:
- Any other relevant information (e.g. version of related package dependencies installed):

Did you build using source code or from docker file? If from source code, please provide the below information:
- Python Version:
- Node Version:
- Redis Version:
- Mongo Version: 
- Screen Resolution:
```

**Comment by kimeetok** [comment]
Hi @simin-ong , could I check if you built AI Verify from source code or Dockerfile? 

There are some known issues using the Docker version of the API connector if you are on WSL, so if you are using Docker AI Verify, we recommend using an Ubuntu Virtual Machine. If that is not possible, you could build AI Verify from source code instead. https://imda-btg.github.io/aiverify/getting-started/source-code-setup/

**Comment by kimeetok** [comment]
Hi @simin-ong this should be fixed by our latest [v0.10.1](https://github.com/IMDA-BTG/aiverify/releases/tag/v0.10.1) release :)
